.env

```diff
@@ -6,3 +6,4 @@ EXPO_PUBLIC_API_URL=https://paas-mpa-backend.staging.bratislava.sk
 EXPO_PUBLIC_MINIO_BUCKET=paas-mpa-dev
 EXPO_PUBLIC_DEPLOYMENT=development
 EXPO_PUBLIC_SENTRY_DNS=https://b58eb0384ab7010886ff5cae0417cad5@o701870.ingest.us.sentry.io/4508025288065024
+EXPO_PUBLIC_CAPTCHA_SITE_KEY=0x4AAAAAAAGcjNLteOzO0fDF
```

Comment:
Imho, it's better to have envs like `PLAUSIBLE_KEY` vs `ANALYTICS_KEY`, so I would suggest `TURNSTILE_SITE_KEY` instead of `CAPTCHA_SITE_KEY`